rfd/0101-pod-rbac.md

```diff
@@ -1,6 +1,6 @@
 ---
 authors: Tiago Silva (tiago.silva@goteleport.com)
-state: draft
+state: implemented
```

Comment:
```suggestion
state: implemented (v12.0.1)
```

It's nice to put the initial version this was released in too, so as we look back at old RFDs we know when the functionality was introduced.